eng/performance/benchmark_jobs.yml

```diff
@@ -106,7 +106,7 @@ jobs:
             ${{ each framework in parameters.frameworks }}:
               ${{ framework }}:
                 _Framework: ${{ framework }}
-                _Configs: CompilationMode=Tiered
+                _Configs: CompilationMode=Tiered;RunKind=${{ parameters.kind }}
```

Comment:
Will this enforce a requirement that these never have spaces in them? I don't think we have any that do today but who knows what someone would type in the future?

Reply:
I don't believe so. Do we have the equivalent of String.Replace in YAML? Are you thinking that we should quote this string?

Reply:
Just asking - seems like the kind of thing that might bite us later, but I don't have a specific concern.